projects/cashmere/src/lib/datepicker/calendar-body/calendar-body.component.html

```diff
@@ -11,8 +11,6 @@
         aria-hidden="true"
         class="hc-calendar-body-label"
         [attr.colspan]="_firstRowOffset"
-        [style.paddingTop]="_cellPadding"
-        [style.paddingBottom]="_cellPadding"
```

Comment:
I saw this happening in Patient Safety Monitor so just adding the fix to this PR.  This padding on the top and bottom of the spacer was making an abnormally large first row.
<img width="327" alt="Screen Shot 2019-10-10 at 12 46 46 PM" src="https://user-images.githubusercontent.com/22795893/66597361-0f6f9400-eb5c-11e9-94c8-98508ace0a4e.png">